pkg/crinit/aggregated/init.go

```diff
@@ -111,15 +122,29 @@ func newSubCmdInit(cmdOut io.Writer, pathOptions *clientcmd.PathOptions,
 }
 
 // validateOptions ensures that options are valid.
-func validateOptions(opts *aggregatedClusterRegistryOptions) error {
+func validateOptions(opts *aggregatedClusterRegistryOptions, cmdOut io.Writer) error {
 	opts.APIServerServiceType = v1.ServiceType(opts.apiServerServiceTypeString)
+	if len(opts.AggContext) == 0 {
+		glog.Warningf(
```

Comment:
I'm not sure this is worth a warning: it seems like a common use case that people will want to aggregate to the same context that the pod is running in.